frostsnapp/lib/wallet_tx_details.dart

```diff
@@ -735,7 +727,7 @@ Widget buildDetailsColumn(
               ),
             ],
           ),
-          title: fee == null ? Text('Unknown') : SatoshiText(value: fee),
```

Comment:
Review TODO test this, i suspect this will show zero when we probably want Unknown or similar